### PR TITLE
seperated cluster config from launch script

### PR DESCRIPTION
**Pull request by karthikeyan-cere**

### CODE REVIEW

File: scripts/network-launch/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
 # Launch Network
 
 Scripts to launch a Network.
+
+```bash
+$ bash launch.sh --file=<CLUSTER-CONFIG-PATH>
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/README.md:6]: Rename the parameter to `--cluster`

File: scripts/network-launch/cluster-config/dev.txt [hunk 2]
```diff
@@ -0,0 +1,12 @@
+bootNodeIP="164.90.173.129"
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/cluster-config/dev.txt:1]: Rename folder `cluster-config` to `clusters`.

Comment (AndreiNavoichyk) [reply scripts/network-launch/cluster-config/dev.txt:1]: Rename `.txt` everywhere. Just `dev`, `dev1`, etc.

Comment (karthikeyan-cere) [reply scripts/network-launch/cluster-config/dev.txt]: updated.

File: scripts/network-launch/README.md (hunk 1 above)
Comment (karthikeyan-cere) [reply scripts/network-launch/README.md]: updated

File: scripts/network-launch/clusters/ext-dev-1 [hunk 3]
```diff
@@ -0,0 +1,12 @@
+bootNodeIP="143.198.145.104"
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/clusters/ext-dev-1:1]: Rename ext-dev-1 -> ext-devs-1. And all other

File: scripts/network-launch/clusters/dev [hunk 4]
```diff
@@ -0,0 +1,12 @@
+bootNodeIP="164.90.173.129"
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/clusters/dev:1]: Rename dev -> devnet

File: scripts/network-launch/clusters/dev1 [hunk 5]
```diff
@@ -0,0 +1,12 @@
+bootNodeIP="164.90.155.170"
```
Comment (AndreiNavoichyk) [inline scripts/network-launch/clusters/dev1:1]: Rename dev1 to devnet1

**Review by AndreiNavoichyk — APPROVED** [review]

File: scripts/network-launch/clusters/ext-dev-1 (hunk 3 above)
Comment (karthikeyan-cere) [reply scripts/network-launch/clusters/ext-dev-1]: renamed

File: scripts/network-launch/clusters/dev1 (hunk 5 above)
Comment (karthikeyan-cere) [reply scripts/network-launch/clusters/dev1]: renamed

File: scripts/network-launch/clusters/dev (hunk 4 above)
Comment (karthikeyan-cere) [reply scripts/network-launch/clusters/dev]: renamed

**Pull request merged**